src/custom-conversion/validateConfig.spec.ts

```diff
@@ -1,10 +1,14 @@
+import { describe, it } from 'node:test'
+import assert from 'node:assert/strict'
 import { validateConfig } from './validateConfig.js'
 
-describe('validateConfig()', () => {
-	it('mark invalid config', () =>
-		expect(
+void describe('validateConfig()', () => {
+	void it('mark invalid config', () => {
+		assert.strictEqual(
```

Comment:
```suggestion
		assert.equal(
```

Strict is implied through `'node:assert/strict'`